src/IO/H5/Python/CombineH5.py

```diff
@@ -36,15 +36,34 @@
     ),
     help="combined output filename (omit file extension)",
 )
+@click.option(
+    "--check_src",
+    "-c",
+    required=True,
+    type=bool,
+    help=(
+        "flag to check src files, True implies src files exist and can be"
+        " checked, False implies no src files to check."
+    ),
+)
+def combine_h5_command(file_prefix, subfile_name, output, check_src):
+    """Combines multiple HDF5 volume files"""
```

Comment:
the first line and the rest should be one string (remove `"""` at the end of this line)